ci/test_wheel.sh

```diff
@@ -28,5 +28,5 @@ else
        --import-mode=append \
        --benchmark-disable \
        -k "not test_property_graph_mg and not test_bulk_sampler_io" \
-       ./python/${package_name}/${python_package_name}/tests
+       ./python/"${package_name}"/"${python_package_name}"/tests
```

Comment:
```suggestion
       "./python/${package_name}/${python_package_name}/tests"
```

Since there isn't anything else here like wildcards that need to be evaluated, I have a mild preference for just wrapping the entire thing in double quotes, instead of having 2 inner sets of double quotes. Think that's a little easier to read and to modify.